kratos/solving_strategies/strategies/explicit_solving_strategy.h

```diff
@@ -98,7 +98,6 @@ class ExplicitSolvingStrategy
         // Validate and assign defaults
         ThisParameters = this->ValidateAndAssignParameters(ThisParameters, this->GetDefaultParameters());
         this->AssignSettings(ThisParameters);
-        mpExplicitBuilder = Kratos::make_unique<ExplicitBuilder<TSparseSpace, TDenseSpace>>();
```

Comment:
you sure abt this change?
It doesn't seem right

Reply:
This class is not supposed to be touched, let me  check

Reply:
It is correct, it is created later, using the strategies